.github/workflows/test.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version: ["3.11"]
-        os: [ubuntu-22.04, macos-12]
+        os: [ubuntu-22.04]
```

Comment:
Why was macos-12 removed?

Reply:
Not sure - this was in @weiji14 's modifications in the OS-X branch

Reply:
Maybe a bad merge conflict? I didn't remove `macos-12` though, actually added `macos-14`.